ods_ci/tests/Resources/Files/pipeline-samples/v2/cache-disabled/ray_integration.py

```diff
@@ -24,7 +24,7 @@ def ray_fn() -> int:
             worker_cpu_limits=1,
             worker_memory_requests=1,
             worker_memory_limits=2,
-            image="quay.io/modh/ray:2.35.0-py39-cu121",
+            image="quay.io/modh/ray@sha256:0d715f92570a2997381b7cafc0e224cfa25323f18b9545acfd23bc2b71576d06",
```

Comment:
This sha belongs to `quay.io/modh/ray:2.35.0-py39-cu121`. We should use the sha for `quay.io/modh/ray:2.35.0-py311-cu121"`

Reply:
Unfortunately, it is necessary to use Python 3.9 because in interactive mode both server and client must have the same minor Python version and pipelines are currently tested with Python 3.9 images.